sleap/gui/app.py

```diff
@@ -1364,15 +1364,15 @@ def updateStatusMessage(self, message: Optional[str] = None):
                     )
                     message += " in video"
 
-            lf = self.state["labeled_frame"]
-            n_instances = 0 if lf is None else len(lf)
+            visible_instances = self.state["labeled_frame"].instances_to_show
+            n_instances = 0 if visible_instances is None else len(visible_instances)
             message += f"{spacer}Current frame: {n_instances} instances"
-            if not self.state["show instances"]:
+            if n_instances and not self.state["show instances"]:
```

Comment:
**Nit:** When using integers or other numeric types in a conditional expression, I find it more readable to explicitly do the binary comparison (`n_instances > 0`) than to rely on the implicit boolean casting (`bool(n_instances) == False` ~= `bool(0) == False`).

(PS: "nit" == a nit-picky suggestion based on style preference that is not mandatory.)

Reply:
The nit was well-taken. Lets not rely on implicit casting. This line has been changed to the suggested `n_instances > 0`.